src/components/Views/CameraView/CameraView.js

```diff
@@ -204,7 +204,11 @@ class CameraView extends React.Component<
         {fromEditor && (
           <TouchableOpacity
             style={styles.cancelButton}
-            onPress={() => this.props.navigation.goBack()}
+            onPress={() =>
```

Comment:
Won't this mess up the back button/stack on the navigation?

Reply:
For some reason, it's not going back to the editor from goBack hmm

Reply:
What happens when you goBack?

Reply:
It ends up going to the camera view. So the flow here is:
1. create observation from camera
2. select category
3. in editor, open camera
4. hit cancel
5. currently, it takes you back to the camera view instead of to the editor